README.md

```diff
@@ -49,31 +49,145 @@ print line : { operation : 1 plus 2 }
+        </tr>
+        <tr>
+            <th>General Release</th>
+            <td>1.0</td>
+            <td>August 2016</td>
+            <td>Public announcement, glory to everyone.</td>
+        </tr>
+    </tbody>
+</table>
 
 ## Intentionally Missing Items
 
 No language is perfect. The following are some seemingly obvious omissions in GLS that are due to languages not supporting them:
 
-| Feature               | C# | Java      | Python    | Ruby      | TypeScript  |
-|-----------------------|----|-----------|-----------|-----------|-------------|
-| Enums Without Values  |    |           |           | *Missing* |             |
-| Foreach Over Values   |    |           |           |           |  *Missing*  |
-| Multiline Lambdas     |    |           | *Missing* |           |             |
-| Optional Parameters   |    | *Missing* |           |           |             |
-| Overloaded Functions  |    |           | *Missing* | *Missing* |  *Missing*  |
-| String.Replace        |    |           |           |           |  *Abnormal* |
-| Switch Statements     |    |           | *Missing* |           |             |
+<table>
+    <thead>
+        <th>Feature</th>
+        <th>C#</th>
+        <th>Java</th>
+        <th>Python</th>
+        <th>Ruby</th>
+        <th>TypeScript</th>
+    </thead>
+    <tbody>
+        <tr>
+            <th>Default Member Variable Values</th>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td>*Missing*</td>
+            <td></td>
+        </tr>
+        <tr>
+            <th>Enums Without Values</th>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td>*Missing*</td>
+            <td></td>
+        </tr>
+        <tr>
+            <th>Foreach Over Values</th>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td>*Missing*</td>
+        </tr>
+        <tr>
+            <th>Multiline Lambdas</th>
+            <td></td>
+            <td>*Missing*</td>
+            <td></td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <th>Optional Parameters</th>
+            <td></td>
+            <td>*Missing*</td>
+            <td></td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <th>Overloaded Functions</th>
+            <td></td>
+            <td></td>
+            <td>*Missing*</td>
+            <td>*Missing*</td>
+            <td>*Missing*</td>
+        </tr>
+        <tr>
+            <th>String.Replace</th>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td>*Abnormal*</td>
+        </tr>
+        <tr>
+            <th>Switch Statements</th>
+            <td></td>
+            <td></td>
+            <td>*Missing*</td>
+            <td></td>
+            <td></td>
+        </tr>
+    </tbody>
+</table>
```

Comment:
Huh. Aren't these README tables already converted? I thought I merged that into master a couple hours ago